cli/command/system/info.go

```diff
@@ -319,6 +319,13 @@ func prettyPrintInfo(dockerCli *command.DockerCli, info types.Info) error {
 
 	fmt.Fprintf(dockerCli.Out(), "Live Restore Enabled: %v\n", info.LiveRestoreEnabled)
 
+	if info.ProxyEnv != nil && len(info.ProxyEnv) > 0 {
```

Comment:
Yes I do, because `len([]string{}) == 0` and we don't want to print the information if there is none to show.